internal/services/containerapps/helpers/container_apps.go

```diff
@@ -404,7 +404,7 @@ func ContainerDaprSchema() *pluginsdk.Schema {
 
 				"app_port": {
 					Type:        pluginsdk.TypeInt,
-					Required:    true,
+					Required:    false,
```

Comment:
Better be `Optional: true`

Reply:
I replaced Required: false with Optional: true, thanks